src/zenml/cli/model.py

```diff
@@ -11,650 +11,507 @@
-            metadata: Filter models by Metadata (Tags).
-        """
-        metadata = dict(metadata) if metadata else None
-        registered_models = model_registry.list_models(metadata=metadata)
-        # Print registered models if any
-        if registered_models:
-            cli_utils.pretty_print_registered_model_table(registered_models)
-        else:
-            cli_utils.declare("No models found.")
-
-    @models.command(
-        "register",
-        help="Register a model with the active model registry.",
-    )
-    @click.argument(
-        "name",
-        type=click.STRING,
-        required=True,
-    )
-    @click.option(
-        "--description",
-        "-d",
-        type=str,
-        default=None,
-        help="Description of the model to register.",
-    )
-    @click.option(
-        "--metadata",
-        "-t",
-        type=(str, str),
-        default=None,
-        help="Metadata or Tags to add to the model. can be used like: -m key1 value1 -m key2 value",
-        multiple=True,
-    )
-    @click.pass_obj
-    def register_model(
-        model_registry: "BaseModelRegistry",
-        name: str,
-        description: Optional[str],
-        metadata: Optional[Dict[str, str]],
-    ) -> None:
-        """Register a model with the active model registry.
-
-        Args:
-            model_registry: The model registry stack component.
-            name: Name of the model to register.
-            description: Description of the model to register.
-            metadata: Metadata or Tags to add to the registered model.
-        """
-        try:
-            model_registry.get_model(name)
-            cli_utils.error(f"Model with name {name} already exists.")
-        except KeyError:
-            pass
-        metadata = dict(metadata) if metadata else None
-        model_registry.register_model(
+
+@model.command("register", help="Create a model.")
+@click.option(
+    "--name",
+    "-n",
+    help="The name of the model.",
+    type=str,
+    required=True,
+)
+@click.option(
+    "--license",
+    "-l",
+    help="The license under which the model is created.",
+    type=str,
+    required=False,
+)
+@click.option(
+    "--description",
+    "-d",
+    help="The description of the model.",
+    type=str,
+    required=False,
+)
+@click.option(
+    "--audience",
+    "-a",
+    help="The target audience for the model.",
+    type=str,
+    required=False,
+)
+@click.option(
+    "--use-cases",
+    "-u",
+    help="The use cases of the model.",
+    type=str,
+    required=False,
+)
+@click.option(
+    "--tradeoffs",
+    help="The tradeoffs of the model.",
+    type=str,
+    required=False,
+)
+@click.option(
+    "--ethical",
+    "-e",
+    help="The ethical implications of the model.",
+    type=str,
+    required=False,
+)
+@click.option(
+    "--limitations",
+    help="The known limitations of the model.",
+    type=str,
+    required=False,
+)
```

Comment:
Actually, aren't all of these simply tags? If so, are they handled in a special way in the UI? Or why are they separate args?

Reply:
Yeah, they will be represented in the main tab of the model and model version separately, so they are structured tags in a sense.